{{ cookiecutter.__project_name_kebab }}/.github/workflows/publish.yml

```diff
@@ -33,9 +33,7 @@ jobs:
         run: python -m flit build
 
       - name: Publish to PyPI
-        env:
-          FLIT_USERNAME: '__token__'
-          FLIT_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
-          FLIT_INDEX_URL: https://upload.pypi.org/legacy/
-        run: python -m flit publish
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
suggestion: while you're here, perhaps use Python 3.11?

other than that 👍🏼

Reply:
ok, I take it back. the job needs

```
```
permissions:
      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
```

as per https://github.com/pypa/gh-action-pypi-publish

This means it is best to separate the build job from the release one because of the permission on the job

Reply:
Yes. You're right. I missed not only the `id-token: write` permission, but also the "publish" environment referenced in the docs :facepalm:

Reply:
oh yeah. I am still a bit wary about doing it all in one job because of the id-token:write permission, but happy to be convinced otherwise